components/Branding/ColorSection/ColorSection.tsx

```diff
@@ -3,7 +3,9 @@ import Swatch from 'components/Branding/Swatch/Swatch';
 import Content from 'components/Content/Content';
 
 function ColorSection() {
+  // @ts-expect-error
```

Comment:
it's fine to have error expects in test code, but not in UI code... what's the error here?

Reply:
So, the ts error is: "Property 'primary' does not exist on type '{}'.ts(2339)", and the error flag is on `brandColorsObject.primary`. I think I found a way to solve the error.

<img width="1440" alt="Screenshot 2024-03-02 at 8 44 40 AM" src="https://github.com/OperationCode/front-end/assets/46408716/d0dbbef2-17b5-499b-a806-8ef4aa54e6cc">